tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_SphericalCompression.cpp

```diff
@@ -549,16 +546,13 @@ SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.SphericalCompression",
   ERROR("Failed to trigger ERROR in an error test");
 }
 
-// [[OutputRegex, Target radius]]
 [[noreturn]] SPECTRE_TEST_CASE(
```

Comment:
while squashing, could you combine this test case with the one on line 468, maybe just sticking the code into a free function and passing in the generator? Reduces the number of tests helps keep test runtime down :)

Reply:
Turns out I don't need the if logic anymore (that was for when it was an error test), so I just added both tests to the test on line 468. Since it's just two lines now, I figure no need for a free function